### PR TITLE
card page

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot (83)
Screenshot (84)
Screenshot (85)

### CODE REVIEW

**Comment by TheAmlanSaikia** [comment]
Looks good to me. I assume p and m stands for padding and margin, right?

**Review by TheAmlanSaikia — APPROVED** [review]
Looks good to me. I assume p and m stands for padding and margin, right?

**Comment by aniketsingh13** [comment]
> Looks good to me. I assume p and m stands for padding and margin, right?

**Comment by aniketsingh13** [comment]
> Looks good to me. I assume p and m stands for padding and margin, right?

yes

File: Styles/card.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+cursor: pointer;
+}
+.card-cart:hover{
+    background-color: var(--primary-color);
+}
+.card-whislist{
+    color: var(--backgournd-color);
+background-color: var(--secondary-hover);
+cursor: pointer;
+}
```
Comment (rohanmathur91) [inline Styles/card.css]: Indentation is not proper.

**Review by rohanmathur91 — APPROVED** [review]
Looks good.

File: Styles/card.css (hunk 1 above)
Comment (aniketsingh13) [reply Styles/card.css]: will change that

**Pull request merged**